packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js

```diff
@@ -341,7 +341,7 @@ describe('ReactDOMFiberAsync', () => {
       ReactDOM.render(<Component />, container);
       // Update
       expect(() => ReactDOM.render(<Component />, container)).toThrow(
-        'flushSync was called from inside a lifecycle method',
+        'flushSync was called from inside a lifecycle method. It cannot be called when React is already rendering.',
```

Comment:
In general it's fine if we don't test the full message

Reply:
Ah, fine, just want keep the consistent

Reply:
Wii remove this this evening

Reply:
@gaearon done~

Reply:
@gaearon done~